### ISSUE TITLE
[Bug]: Can't download the browser

### ISSUE THREAD

**Issue by Allamaris0** [CheshireCaat/browser-with-fingerprints#39]
**Describe the bug**
It downloads  only:
![image](https://github.com/user-attachments/assets/5f753f7a-7a19-4198-b154-fd33747f555c)

I get error:
```
\node_modules\browser-with-fingerprints\src\plugin\connector\index.js:48
            () => reject(new Error(`Timed out while calling the "${name}" method.`)),
                         ^

Error: Timed out while calling the "fetch" method.
    at Timeout._onTimeout (\node_modules\browser-with-fingerprints\src\plugin\connector\index.js:48:26)
    at listOnTimeout (node:internal/timers:594:17)
    at process.processTimers (node:internal/timers:529:7)

Node.js v22.12.0
```

I tried puppeteer-with-fingerprints and playwright-with-fingerprints

**Comment by CheshireCaat** [comment]
You can increase global timeout using the `setRequestTimeout` method, or download engine manually - information about this can be found in documentation and issues.

**Comment by Allamaris0** [comment]
`setRequestTimeout doesn't` help. I downloaded FastExecuteScriptProtected.x64.zip manually. Could you give me link with instruction how to unpack it? Because I can't find.

**Comment by CheshireCaat** [comment]
You don’t need to unpack it manually, just place it inside `data/engine/28.1.2` folder.

**Comment by Allamaris0** [comment]
Yes, I tried it, but it doesn't work. When `FastExecuteScript.x64.zip` is inside data/engine/28.1.2 then it try to download. When I remove it and I put `FastExecuteScriptProtected.x64.zip` then it wants to install the browser but there is error that no FastExecuteScript.x64.zip in data/engine/28.1.2.

**Comment by CheshireCaat** [comment]
You should rename zip - it must be named as shown on your screenshot.

Try to completely remove previous archive before moving a new one.

**Comment by Allamaris0** [comment]
It doesn't help. I wrote if `FastExecuteScript.x64.zip` is inside data/engine/28.1.2 then it tries to download the browser and then I got the error with fetching.

**Comment by CheshireCaat** [comment]
Please attach code example and all steps to reproduce your problem, without it I can not help.

**Comment by Allamaris0** [comment]
Just simple code
